code-submissions-clustering-core/src/test/kotlin/org/jetbrains/research/code/submissions/clustering/DistanceCalculationTest.kt

```diff
@@ -0,0 +1,126 @@
+package org.jetbrains.research.code.submissions.clustering
```

Comment:
It is also should be moved into org.jetbrains.research.code.submissions.clustering.load.distance package